roles/openshift-imagetag/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Setting target_namespace 
+  set_fact: 
+    target_namespace: "{{ target_namespace | default('') }}"
+
+- name:  Tagging as {{ img_tag }} on a namespace {{ target_namespace }}
+  command: >
+    oc tag --insecure=true  {{ source_img }} {{ target_namespace }}/{{ img_tag }}
```

Comment:
Please remote the `--insecure=true` portion of this command.  Also, the `target_namespace` will have the `-n` option set (if the variable is set), so need to keep that separate from the actual `img_tag` - i.e.: something like `{{ img_tag }} {{ target_namespace }}`.